### ISSUE TITLE
Tableview for mobile

### ISSUE THREAD

**Issue by dbauszus-glx** [GEOLYTIX/xyz#115]
While not that practical it is not insanely hard to create a tableview tab for the mobile view. Especially in consideration of location tables which otherwise would have to be filtered from the location view.

**Comment by cityremade** [comment]
do we want one?

**Comment by dbauszus-glx** [comment]
Yes. In the case of single column tables we would like to be able to enable these in a custom mobile view. For example for the selection of sites.
